#! /usr/bin/env python
# Task Long configuration

PACKAGE='sift_detect_holo'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name                    Type            Description
#       Default    Min   Max
gen.add("siftThreshold",  double_t, 0,    "min distance to keep a pair of sift points",  5.0, 0.0, 10.0)


exit(gen.generate(PACKAGE, "sift_detect_holo", "SiftDescript"))
